src/backend/common/UsageTracker.h

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 The NXT Authors
```

Comment:
That's a good point I forgot about. In that case I think I want to have two usage tracker classes. I want to be able to use the usage tracker classes by themselves (without resources) in the command buffer state tracker (for tracking usages locally).